src/controller/SetUpCodePairer.cpp

```diff
@@ -78,7 +78,7 @@ CHIP_ERROR SetUpCodePairer::Connect(SetupPayload & payload)
         {
             isRunning = true;
         }
-        VerifyOrReturnError(searchOverAll || CHIP_NO_ERROR == err, err);
+        VerifyOrReturnError(searchOverAll || CHIP_NO_ERROR == err || CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE == err, err);
```

Comment:
Should the same change be done on line 90 for softap as I imagine softap may also not be supported?

A comment would help explaining why would help.
Specifically I would have expected `searchOverAll` to mean that all things are tried but it seems instead the payload is specific (ble) yet we still want to do ip discovery.